tests/test_port_add_remove.py

```diff
@@ -119,7 +119,7 @@ def test_remove_add_remove_port_with_buffer_cfg(self, dvs, testlog):
 
         num = asic_db.wait_for_n_keys("ASIC_STATE:SAI_OBJECT_TYPE_PORT",
                                       num_of_ports-1,
-                                      polling_config = PollingConfig(polling_interval = 1, timeout = 5.00, strict = False))
+                                      polling_config = PollingConfig(polling_interval = 1, timeout = 10.00, strict = False))
```

Comment:
Why increase the timeout here? Can you elaborate why this dhcp rate limiting change needs increased timeout for these test cases?

Reply:
The increased initialization time was set after analyzing test failures. A 30-second timeout was chosen as a safe limit to ensure the DVS is fully ready before running the tests.i have reduced the timeout to 10-second